resources/css/InaSAFE.css

```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: 'Ubuntu', 'Lucida Grande', 'Segoe UI', sans-serif;
+  font-family: 'Ubuntu', 'Lucida Grande', 'Segoe UI', 'Arial', sans-serif;
```

Comment:
Maybe you should give hight priority to Arial? Can you get someone to confirm if this fixes it?